lib/appmap/command/agent_setup/init.rb

```diff
@@ -11,32 +10,19 @@ module AgentSetup
 
       class Init < InitStruct
         def perform
-          if File.exist?(config_file)
-            puts AppMap::Util.color(%(The AppMap config file #{config_file} already exists.), :magenta)
-            return
-          end
-
-          ensure_directory_exists
-
           config = {
             'name' => Service::Guesser.guess_name,
             'packages' => Service::Guesser.guess_paths.map { |path| { 'path' => path } }
           }
-          content = YAML.dump(config).gsub("---\n", '')
 
-          File.write(config_file, content)
-          puts AppMap::Util.color(
-            %(The following AppMap config file #{config_file} has been created:),
-            :green
-          )
-          puts content
-        end
-
-        private
+          result = {
+            :configuration => {
```

Comment:
The rest of appmap-ruby uses new-style syntax for hashes with symbol keys. Probably best to follow that convention here.